src/cmd/run.go

```diff
@@ -26,17 +26,13 @@ var runCmd = &cobra.Command{
 		if err != nil {
 			log.Fatal(err)
 		}
-		buildMode := types.BuildMode{
-			Type:        types.BuildModeTypeLocal,
-			Mount:       true,
```

Comment:
This confused me as I was trying to figure out if we mounted in production mode and this code made it look like we did.